kt/plugins/godot-intellij-plugin/src/main/kotlin/godot/intellij/plugin/listener/GodotSceneBulkFileListener.kt

```diff
@@ -35,24 +41,29 @@ class GodotSceneBulkFileListener(private val project: Project) : BulkFileListene
             }
     }
 
-    private fun initialIndexing() = getContainingFiles()
-        .forEach { vFile ->
+    private fun initialIndexing() = getContainingFiles { vFiles ->
+        vFiles.forEach { vFile ->
             virtualFileChanged(vFile)
         }
+    }
 
     @Suppress("UnstableApiUsage")
-    private fun getContainingFiles() = FileBasedIndex
-        .getInstance()
-        .getContainingFiles(
-            FileTypeIndex.NAME,
-            PlainTextFileType.INSTANCE,
-            GlobalSearchScope.projectScope(project)
-        )
+    private fun getContainingFiles(callback: (List<VirtualFile>) -> Unit) {
```

Comment:
I would rename this method as it now execute a callback

Reply:
:+1: